### DISCUSSION TITLE
Running search with multiple proteases

### DISCUSSION THREAD

**Discussion by OmarArias-Gaguancela** [vdemichev/DiaNN#1019] [General]
Hello. 
Please, I have a few questions regarding one of the settings in DIA-NN. 
My samples for DIA proteomics were digested with a protease mixture of Trypsin and Lys-C. 
I am trying to run my library and samples with DIA-NN, however, in its interface, in the option for protease, There are no options for the protease mixture I have. What should I do?. Should I run the DIA-NN analysis with each protease individually, and then combine the data? Or what would be the best approach? 
Looking forward to hearing your response. 
Thank you,
Best,
Omar Arias-Gaguancela.

**Comment by vdemichev** [comment]
Hi Omar,

For Trypsin + Lys-C please just use the default setting, which is trypsin.

Best,
Vadim

**Reply by OmarArias-Gaguancela** [reply]
Thank you for your reply. 
Best,
Omar.
